content/courses/onchain-development/anchor-pdas.md

```diff
@@ -454,7 +454,49 @@ pub struct MovieAccountState {
 }
 ```
 
-#### 3. Add Movie Review
+For this account struct, we will be implementing the space trait:
```

Comment:
can you add some details of what the space trait is for and why its good to use. the code block comment covers it, but I think having it in the regular paragraph will be more usable by readers

Reply:
Agreed. Added more details regarding the space trait